### PR TITLE
Create validate-dual-stack.md

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/network/validate-dual-stack.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+<!--
+* Kubernetes 1.16 or later
+* Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+* Kubenet network plugin
+* Kube-proxy running in mode IPVS
+* [Dual-stack enabled](/docs/concepts/services-networking/dual-stack/) cluster
+-->
+* Kubernetes 1.16 或更高版本
+* 提供程序对双栈网络的支持 (云提供程序或其他方式必须能够为 Kubernetes 节点提供可路由的 IPv4/IPv6 网络接口)
```
Comment (xichengliudui) [inline content/zh/docs/tasks/network/validate-dual-stack.md]:
```suggestion
* 提供程序对双栈网络的支持 (云供应商或其他方式必须能够为 Kubernetes 节点提供可路由的 IPv4/IPv6 网络接口)
```

File: content/zh/docs/tasks/network/validate-dual-stack.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+* Kubenet 网络插件
+* Kube-proxy 在 IPVS 模式下运行
+* [启用双栈](/docs/concepts/services-networking/dual-stack/) 集群
```
Comment (xichengliudui) [inline content/zh/docs/tasks/network/validate-dual-stack.md]:
```suggestion
* [启用双协议栈](/docs/concepts/services-networking/dual-stack/) 集群
```

File: content/zh/docs/tasks/network/validate-dual-stack.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+```
+ kubectl get svc -l app=MyApp
+NAME         TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+my-service   ClusterIP   fe80:20d::d06b   <none>        80/TCP    9s
+```
+
+<!--
+### Create a dual-stack load balanced Service
+-->
+### 创建双栈负载均衡服务
```
Comment (xichengliudui) [inline content/zh/docs/tasks/network/validate-dual-stack.md]:
```suggestion
### 创建双协议栈负载均衡服务
```

File: content/zh/docs/tasks/network/validate-dual-stack.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+<!--
+### Validate node addressing
+-->
+### 验证节点寻址
+
+<!--
+Each dual-stack Node should have a single IPv4 block and a single IPv6 block allocated. Validate that IPv4/IPv6 Pod address ranges are configured by running the following command. Replace the sample node name with a valid dual-stack Node from your cluster. In this example, the Node's name is `k8s-linuxpool1-34450317-0`:
+-->
+每个双栈节点应分配一个 IPv4 块和一个 IPv6 块。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/network/validate-dual-stack.md]:
```suggestion
每个双协议栈节点应分配一个 IPv4 块和一个 IPv6 块。
```

**Comment by zhangx501** [comment]
/assign @xichengliudui
您好，问题已经改正过来了

**Pull request merged**